packages/mobile/src/screens/edit-track-screen/fields/AccessAndSaleField/PremiumRadioField/PremiumRadioField.tsx

```diff
@@ -156,12 +132,6 @@ export const PremiumRadioField = (props: PremiumRadioFieldProps) => {
           </Text>
         </View>
       ) : null}
-      {!isUsdcUploadEnabled ? (
-        <>
-          <Tag style={styles.comingSoon}>{messages.comingSoon}</Tag>
-          <HelpCallout content={renderHelpCalloutContent()} />
```

Comment:
should this be a `Hint` now? or did we want to remove this one?

Reply:
removed cause `isUsdcUploadEnabled` should just be removed i think